src/components/Article.jsx

```diff
@@ -1 +1,23 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ title, date, text }) => (
+  <div className="box">
+    <h2 className="title" data-cy="title">
+      {title}
+    </h2>
+
+    <p className="subtitle">
+      <time dateTime={date.toISOString().slice(0, 10)} data-cy="date">
+        {date.toLocaleDateString('en-US', {
```

Comment:
avoid big js expression in jsx, create a variable for this value and ISO date above